src/Components/Atoms/Logo/index.tsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import { Box } from "@mui/material";
+
+export const Logo = () => {
+  const handleClickLogo = () => {
+    window.location.assign("/login");
+  };
+
+  return (
+    <Box onClick={handleClickLogo}>
```

Comment:
Should be only svg code, no action needed.

Reply:
consider it done